packages/medical-domain/src/domains/models/medical/datum/cbg.model.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022-2023, Diabeloop
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏`2022-2023` was correct!